### ISSUE TITLE
When waiting for GitHub API, periodically refresh timer 

### ISSUE THREAD

**Issue by xtuchyna** [thoth-station/mi#588]
## Problem statement
Currently, the user is not incrementally informed about the GH API refresh time. This can lead user to more likely interrupt the data extraction process, which is not desired. 

## Proposal description

Refresh timer either each second, or each minute.

## Acceptance Criteria
- [ ] drain the GH API and see if the waiting the is being refreshed
- [ ] test if after interruption the data is saved

**Comment by KPostOffice** [comment]
Seems pretty straight forward. Should basically just wrap the code [here](https://github.com/thoth-station/mi/blob/b64ae33e246dcb97649e04b026a1c4fb05fbae49/srcopsmetrics/github_handling.py#L77) in a while loop right @xtuchyna ?

**Comment by xtuchyna** [comment]
yes you are right @KPostOffice , this was sort of a todo reminder-ish issue so that we do not forget :)
